netbox/circuits/search.py

```diff
@@ -34,7 +34,7 @@ class CircuitTerminationIndex(SearchIndex):
         ('port_speed', 2000),
         ('upstream_speed', 2000),
     )
-    display_attrs = ('circuit', 'site', 'provider_network', 'description')
+    display_attrs = ('circuit', '_site', '_provider_network', 'description')
```

Comment:
We need to ensure that all scope types are supported (not just site and provider network). To do this, we can reference the `termination` GFK rather than the individual ForeignKey assignments.

```suggestion
    display_attrs = ('circuit', 'termination', 'description')
```

This will return whichever type of related object is assigned:

![screenshot](https://github.com/user-attachments/assets/02bab88c-01a0-4656-9a6e-b235e5fa330f)

Reply:
Agreed, that's much nicer.